CHANGELOG.md

```diff
@@ -80,6 +80,16 @@ New modules
 Additions to existing modules
 -----------------------------
 
+* In `Algebra.Consequences.Base`:
+  ```agda
+  module Congruence (cong : Congruent₂ _≈_ _∙_) (refl : Reflexive _≈_)
```

Comment:
This is not particularly informative -- what is in that module? Same below.

Reply:
I think I've never understood how much detail, and of what character, should be included in `CHANGELOG` when adding a new submodule:
* (show) the module contents themselves  ('types document themselves')
* (tell) informal description of what that contents is, but without detail...

Meta-question: what belongs in a `CHANGELOG` (entry) anyway? 
(I often used to end up being too verbose... and now I seem to have overcompensated in the other direction)